orderer/common/channelparticipation/restapi.go

```diff
@@ -70,13 +70,84 @@ func NewHTTPHandler(config localconfig.ChannelParticipation, registrar ChannelMa
 		router:    mux.NewRouter(),
 	}
 
+	// swagger:operation GET /channels/{channelID} channels listChannel
+	// ---
+	// summary: Returns detailed channel information for an Ordering Service Node (OSN).
+	// parameters:
+	// - name: channelID
+	//   in: path
+	//   description: Channel ID
+	//   required: true
+	//   type: string
+	// responses:
+	//    "200":
+	//       description: Successfully retrieved channel.
+	//       schema:
+	//         "$ref": "#/definitions/channelInfo"
+
 	handler.router.HandleFunc(urlWithChannelIDKey, handler.serveListOne).Methods(http.MethodGet)
 
+	// swagger:operation DELETE /channels/{channelID} channels deleteChannel
```

Comment:
```suggestion
	// swagger:operation DELETE /channels/{channelID} channels removeChannel
```

Reply:
fyi this is only a operationID used to identify each unique operation but otherwise this is only naming 🧹